test/integration/full-content/fixtures/core__audio.parsed.json

```diff
@@ -5,12 +5,14 @@
             "align": "right"
         },
         "innerBlocks": [],
-        "innerHTML": "\n<figure class=\"wp-block-audio alignright\">\n    <audio controls=\"\" src=\"https://media.simplecast.com/episodes/audio/80564/draft-podcast-51-livePublish2.mp3\"></audio>\n</figure>\n"
+        "innerHTML": "\n<figure class=\"wp-block-audio alignright\">\n    <audio controls=\"\" src=\"https://media.simplecast.com/episodes/audio/80564/draft-podcast-51-livePublish2.mp3\"></audio>\n</figure>\n",
+        "innerContent": [ "\n<figure class=\"wp-block-audio alignright\">\n    <audio controls=\"\" src=\"https://media.simplecast.com/episodes/audio/80564/draft-podcast-51-livePublish2.mp3\"></audio>\n</figure>\n" ]
```

Comment:
I'm guessing that these were manually edited?

It's meant to be generated via `npm run fixtures:regenerate`, which on master is producing local changes because of stylistic differences of the generated result.

Reply:
#12124

Reply:
yeah that's right @aduth - I was unaware of `fixtures:regenerate` when I wrote this